server/moodle/mod/livequiz/quizcreator/styles.css

```diff
@@ -0,0 +1,155 @@
+.Modal_div {
+    width: 80%;
+    height: 80%;
+    color: grey;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    overflow: auto;
+    background: white; /*Funny collor  name*/
+    display: flex;
+    flex-direction: column;
+    justify-content: center; /* Centrerer elementerne vertikalt */
+    align-items: center; /* Centrerer elementerne horisontalt */
+    border: 3px solid black;
+    border-radius: 25px; /* Rounded corners */
+    box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.5); /* Shadow effect */
+ 
+ 
+ }
+ 
+  
+ .all_answers_for_question_div {
+    max-height: 50%;
+    max-width: 60%;
+    width: 60%;
+    height: 50%;
+    padding-left: 5%;
+    display: flex;
+    flex-wrap: wrap; /* Allows items to wrap onto the next line if needed */
+    justify-content: flex-start; /* Aligns items to the start of the container */
+ }
+ 
+
+ .container_for_new_answer {
+    width: 40%;
+    height: 15%;
+    max-height: 15%;
+    max-width: 40%;
+ 
+ 
+ }
+ 
+ 
+ .question_input {
+    background: white;
+    width: 85%;
+    height: 20%;
+    border: 2px solid black;
+    border-radius: 2px;
+    box-shadow: 1px 3px 5px rgba(0, 0, 0, 0.5);
+    font-size: large;
+    text-align: center;
+ }
+ 
+ 
+ .file_upload_container {
+    margin-bottom: 20px;
+ }
+ 
+ 
+ .file_upload_input {
+    border: 1px solid #ccc;
+    padding: 8px;
+    border-radius: 5px;
+    font-size: 1rem;
+ }
+ 
+ 
+ .file_upload_preview {
+    display: block;
+    margin-top: 10px;
+    max-width: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+ }
+ .discard_question_button {
```

Comment:
Yet another file where the comments suddenly appear. Please try to make files uniform.